_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Bitcoin Optech Newsletter #56'
+permalink: /en/newsletters/2019/07/24/
+name: 2019-07-24-newsletter
+slug: 2019-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes progress on signet and an idea for just-in-time
+routing on LN.  Also included are our regular sections about bech32
+sending support and notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.1 release candidates:** this upcoming
+  maintenance release fixes several bugs that affected certain RPC
+  commands and caused unnecessarily high CPU use in certain cases.
+  Production users are encouraged to test the current [release
+  candidate][core rc] to ensure that it operates as expected.
+
+## News
+
+- **Progress on signet:** signet is a testnet alternative where valid
+  blocks are signed by a trusted authority.  The authority generally produces
+  a regular series of blocks but may occasionally produce forks leading
+  to block chain reorganizations.  This avoids commonly-encountered
+  problems on testnet such as too-fast block production, too-slow block
+  production, and reorganizations involving thousands of blocks.
+  Subsequent to our earlier report on signet in [Newsletter #37], its
+  author has created an implementation, documented it on a [wiki
+  page][signet wiki], opened a [PR][Bitcoin Core #16411] proposing to
+  add support for signet to Bitcoin Core, and posted a [draft
+  BIP][signet bip post] to the Bitcoin-Dev mailing list.
+  The proposed implementation also makes it easy for teams to create
+  their own independent signets for specialized group testing, e.g.
+  signet author Kalle Alm reports that "someone is already working on a
+  signet with [bip-taproot][] patched on top of it."
+  Signet has the
+  potential to make it much easier for developers to test their
+  applications in a multi-user environment, so we encourage all current
+  testnet users and anyone else interested in signet to review the
+  above code and documentation to ensure signet will fulfill your needs.
+
+- **Additional Just-In-Time (JIT) LN routing discussion:** in the JIT
+  discussion described in the [newsletter][Newsletter #54] two weeks ago,
+  contributor ZmnSCPxj explained why routing nodes needed zero-fee
+  rebalance operations in many cases for JIT routing to be incentive
+  compatible.  This week he [posted][jit with fee] a suggestion to the C-Lightning
+  mailing list on how nodes could be less incentive compatible but still
```

Comment:
As an LN layperson I'm not too sure what "incentive-compatible" means here and why one would want less of it. I would have thought it was a good thing :)